AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5281,6 +5281,7 @@ privacy.madame.de#%#AG_onLoad(function(){if(!document.cookie.includes("euconsent
 coolmath.com#%#//scriptlet('set-cookie', 'GDPR_Reject', 'true')
 johnlewis.com#%#//scriptlet('set-cookie', 'JOHNLEWIS_ENSIGHTEN_PRIVACY_Functional', '1')
 reebok.*#%#//scriptlet('set-cookie', 'notice_preferences', '1')
+gazzetta.it#%#AG_onLoad(function() {var b=new MutationObserver(function() {var c=document.querySelector('#_cpmt-accept'); c&&(b.disconnect(),c.click()) });b.observe(document, {childList: !0,subtree: !0});setTimeout(function() {b.disconnect()}, 1E4)})
```

Comment:
+ condition and subdomain:
`video.gazzetta.it#%#AG_onLoad(function() {if (!document.cookie.includes("euconsent-v2")) {var b=new MutationObserver(function() {var c=document.querySelector('#_cpmt-accept'); c&&(b.disconnect(),c.click()) });b.observe(document, {childList: !0,subtree: !0});setTimeout(function() {b.disconnect()}, 1E4)}})`